src/time/ngage/SDL_systime.cpp

```diff
@@ -171,20 +171,33 @@ bool SDL_GetCurrentTime(SDL_Time *ticks)
 
 bool SDL_TimeToDateTime(SDL_Time ticks, SDL_DateTime *dt, bool localTime)
 {
+    TTime ticks_time;
+    TTime universal_time;
+    TTimeIntervalMicroSeconds offset;
+    TDateTime date_time;
+    TDateTime universal_date_time;
+    TInt64 ticks_div;
+    
     CHECK_PARAM(!dt) {
         return SDL_InvalidParamError("dt");
     }
 
-    // FIXME: Need implementation
-    dt->year = 1970;
-    dt->month = 1;
-    dt->day = 1;
-    dt->hour = 0;
-    dt->minute = 0;
-    dt->second = 0;
-    dt->nanosecond = 0;
-    dt->day_of_week = 4;
-    dt->utc_offset = 0;
+    ticks_time = UnixEpoch();
+    ticks_div = ticks / 1000;
```

Comment:
Note, I'm not a n-gage developer, so this might be a bogus suggestion
```suggestion
    ticks_div = static_cast<TInt64>(ticks) / 1000;
```

Reply:
If you add `[sdl-ci-filter ngage]` to your (last) commit message, ci will only build n-gage.
This cuts ci time, and might speed up feedback time.

Reply:
> If you add `[sdl-ci-filter ngage]` to your (last) commit message, ci will only build n-gage. This cuts ci time, and might speed up feedback time.

Thanks for the tip!

Reply:
I forgot `[sdl-ci-filter]` does not work in pr's, only on your branches :)